packages/@o3r/application/schemas/cms.schema.json

```diff
@@ -49,6 +49,16 @@
           "type": "boolean",
           "description": "Specifies that the blueprint supports theming CSS variables customization",
           "default": true
+        },
+        "ruleFunctionalContentActionEnabled": {
+          "type": "boolean",
+          "description": "Enables the custom rule action of type 'functional content update' (UPDATE_FUNCTIONAL_CONTENT). Should be enabled only if the application implements this custom action type",
+          "default": false
+        },
+        "rulesetMultiscopeSupportEnabled": {
+          "type": "boolean",
+          "description": "Enables multi scope components on rulesets. Should be enabled only if the otter version used by the application supports this feature",
+          "default": true
```

Comment:
The default value seems a bit misaligned with description.
If the default value is `true` that means that the configuration relying on a previous version of Otter (so the previous version of this schema) will not provide this option and will get it enabled per default.
According to the description, it sound to be the opposite that you expect.

Reply:
Totally a mistake, default value is false.